test/integration/operator_policy_error_test.go

```diff
@@ -0,0 +1,193 @@
+			Eventually(
+				common.GetComplianceState(policyName),
+				defaultTimeoutSeconds*2,
+				1,
+			).Should(Equal(policiesv1.NonCompliant))
+			Eventually(
+				common.GetOpPolicyCompMsg(operatorPolicyName),
+				defaultTimeoutSeconds*2,
+				1,
+			).Should(ContainSubstring("the operator namespace ('grcqeoptest-notcreated') does not exist"))
+		})
+
+		It("Namespace "+subscriptionNamespace+" should be created", func() {
+			_, err := common.OcManaged(
+				"create", "namespace", subscriptionNamespace,
+			)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It(operatorPolicyName+" should be patched with correct namespace", func() {
+			_, err := common.OcHub(
+				"patch", "policies.policy.open-cluster-management.io", policyName,
+				"-n", userNamespace, "--type=json", "-p", `[{
+				"op":"replace", 
+				"path":"/spec/policy-templates/0/objectDefinition/spec/subscription/namespace",
+				"value":"grcqeoptest-ns-43568"
+			}]`)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It(operatorPolicyName+" should be patched with incorrect source", func() {
+			_, err := common.OcHub(
+				"patch", "policies.policy.open-cluster-management.io", policyName,
+				"-n", userNamespace, "--type=json", "-p", `[{
+				"op":"replace", 
+				"path":"/spec/policy-templates/0/objectDefinition/spec/subscription/source",
+				"value":"redhat-operators-invalid"
+			}]`)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It(operatorPolicyName+" should be NonCompliant due to incorrect source", func() {
+			By("Checking if the status of the operator policy is NonCompliant")
+			Eventually(
+				common.GetComplianceState(policyName),
+				defaultTimeoutSeconds*2,
+				1,
+			).Should(Equal(policiesv1.NonCompliant))
+			Eventually(
+				common.GetOpPolicyCompMsg(operatorPolicyName),
+				defaultTimeoutSeconds*2,
+				1,
+			).Should(ContainSubstring("CatalogSource 'redhat-operators-invalid' was not found"))
+		})
+
+		It(operatorPolicyName+" should be patched with correct source", func() {
+			_, err := common.OcHub(
+				"patch", "policies.policy.open-cluster-management.io", policyName,
+				"-n", userNamespace, "--type=json", "-p", `[{
+				"op":"replace", 
+				"path":"/spec/policy-templates/0/objectDefinition/spec/subscription/source",
+				"value":"redhat-operators"
+			}]`)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It(operatorPolicyName+" should be patched with incorrect installPlanApproval", func() {
+			_, err := common.OcHub(
+				"patch", "policies.policy.open-cluster-management.io", policyName,
+				"-n", userNamespace, "--type=json", "-p", `[{
+				"op":"replace", 
+				"path":"/spec/policy-templates/0/objectDefinition/spec/subscription/installPlanApproval",
+				"value":"Invalid"
+			}]`)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It(operatorPolicyName+" should be NonCompliant", func() {
+			By("Checking if the status of the operator policy is NonCompliant")
+			Eventually(
+				common.GetComplianceState(policyName),
+				defaultTimeoutSeconds*2,
```

Comment:
Lots of these tests use `defaultTimeoutSeconds*2`. Is that necessary? It would be nice to use the default for most cases, although I understand that some of the tests here might need to wait longer for OLM to create something.